Havit/Havit/Global/UIComponent/ContentsCollectionViewCell.swift

```diff
@@ -0,0 +1,167 @@
+
+class ContentsCollectionViewCell: UICollectionViewCell {
+    static var cellID = "ContentsCollectionViewCell"
+    
+    var mainImgView: UIImageView!
+    var alarmImgView: UIImageView!
+    var titleLabel: UILabel!
+    var subtitleLabel: UILabel!
+    var dateLabel: UILabel!
+    var linkLabel: UILabel!
+    var moreButton: UIButton!
+    var isReadImgView: UIImageView!
+    var alarmLabel: UILabel!
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setUpCell()
+        setAutoLayouts()
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+        setUpCell()
+        setAutoLayouts()
+    }
+    
+    func setUpCell() {
+        mainImgView = {
+            let imgView = UIImageView()
+            imgView.image = UIImage()
+            
+            return imgView
+        }()
+        contentView.addSubview(mainImgView)
+        
+        titleLabel = {
+            let label = UILabel()
+            label.text = "슈슈슉 이것은 제목입니다 슈슉 슉슉 이것"
+            label.numberOfLines = 0
+            
+            return label
+        }()
+        contentView.addSubview(titleLabel)
+        
+        subtitleLabel = {
+            let label = UILabel()
+            label.text = "슈슈슉 이것은 제목입니다 슈슉 슉"
+            
+            return label
+        }()
+        contentView.addSubview(subtitleLabel)
+        
+        dateLabel = {
+            let label = UILabel()
+            label.text = "2021. 11. 24"
+            
+            return label
+        }()
+        contentView.addSubview(dateLabel)
+        
+        linkLabel = {
+            let label = UILabel()
+            label.text = "www.beansbin.oopy.io"
+            
+            return label
+        }()
+        contentView.addSubview(linkLabel)
+        
+        
+        alarmLabel = {
+            let label = UILabel()
+            label.text = "2021. 11. 17 오전 12:30 알림 예정"
+            
+            return label
+        }()
+        contentView.addSubview(alarmLabel)
+        
+        alarmImgView = {
+            let imgView = UIImageView()
+            imgView.image = UIImage()
+            
+            return imgView
+        }()
+        mainImgView.addSubview(alarmImgView)
+        
+        moreButton = {
+            let button = UIButton()
+            button.setTitle("...", for: .normal)
+            return button
+        }()
+        contentView.addSubview(moreButton)
+        
+        isReadImgView = {
+            let imgView = UIImageView()
+            imgView.image = UIImage()
+            
+            return imgView
+        }()
+        contentView.addSubview(isReadImgView)
+        
+    }
+    
+    func setAutoLayouts() {
+        mainImgView.snp.makeConstraints { (make) -> Void in
+            make.top.equalTo(contentView).offset(12)
+            make.leading.equalTo(contentView).offset(16)
+            make.trailing.equalTo(contentView).offset(19)
+            make.height.equalTo(108)
```

Comment:
```suggestion
        mainImgView.snp.makeConstraints {
            $0.top.equalTo(contentView).offset(12)
            $0.leading.equalTo(contentView).offset(16)
            $0.trailing.equalTo(contentView).offset(19)
            $0.height.equalTo(108)
```
P5;
make를 $0으로 쓰는건 어떨까요? 저는 make자체가 큰 뜻이 없는 것 같고 매번 `make in`를 써주는 부분이 복잡해서 $0로 사용하는데, 다른 분들 의견도 들어보고 싶어요.

Reply:
아직 클로저가 많이 익숙하지 않아서 정석대로 표기하고 있는 중이었습니닷
그런데 단축인자로 쓰는게 훨씬 더 간단하고 코드 치기에도 간편한 것 같아요! 😃

Reply:
저도 `make`자체에 큰 뜻이 없어보여 `make`보다 `$0` 단축인자를 사용해 쓰는게 간결하고 읽기 좋은 것 같습니다!